cypress/features/regression/accessibility/accessibilityDesignSystem.feature

```diff
@@ -205,4 +205,15 @@ Feature: Accessibility tests - Design System folder
   @accessibility
   Scenario: Component Toast
     When I open "Design System Toast Test" component page "visual" in no iframe
-    Then "Toast" component has no accessibility violations
\ No newline at end of file
+    Then "Toast" component has no accessibility violations
```

Comment:
in this case we need to delete also `card` test from `accessibilityCommon.feature` file